Documentation/IntroductionPackage/Index.rst

```diff
@@ -51,7 +51,19 @@ To install the Introduction Package run the following command:
 
          ddev composer require typo3/cms-introduction
 
-This command will download and activate the extension.
+Since TYPO3 11.5 it is essential to setup the extension afterwards by running
+the following command:
+
+.. tabs::
+
+   .. group-tab:: bash
+
+       .. code-block:: bash
+
+         composer exec typo3 extension:setup
+
+For older core versions go to the backend extension manager and activate the
```

Comment:
I am not sure that we need this sentence as this documentation is about version 11.5 and not older ones. I suggest to remove it.

Reply:
We will do that in a follow up